README.md

```diff
@@ -4,19 +4,29 @@ Kubernetes operator for deploying Llama applications
 # Tools
 * Kind (`brew install kind`)
 * Kubectl (`brew install kubectl`)
+* Docker
 
 # Getting Started
 
+Bootstrap your system to make it ready to develop.
```

Comment:
sugg: each of the other commands suggest a terminal, leaving this one ambiguous. Wanna push these next to the kind load docker-image commands?

Reply:
this is all just a poc anyway so I will merge it into the term 1 instructions.